co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstanceManager.java

```diff
@@ -91,25 +101,30 @@ public void onNewRegistration(InfrastructureRegistrationMessage registration) {
      * 
      */
     private void newInfrastructureInstance(String infrastructureId, InetAddress rxMessageIpAddress, int rxMessagePort,
```

Comment:
This function name is a bit misleading. It looks like trying to create a new object. The name does not account for adding the object into a global map of instances.

Reply:
This method is not part of my PR and does exactly what it says it does. This is an Infrastructure Instance Manager. A class that managers Infrastructure Instances. This method adds an infrastructure instance to the managed instances. U di bit think it is misleading at all. I could add more documentation to the method but even that is kind of outside my PR since my PR is about adding sensor registration and object detection.